content/documentation/pages/9-applications/1-pre-packaged-applications.md

```diff
@@ -0,0 +1,144 @@
+| [syslog](%stream-applications-doc%/#spring-cloud-stream-modules-syslog-source)                 | [tasklaunchrequest-transform](%stream-applications-doc%/#spring-cloud-stream-modules-tasklaunchrequest-transform-processor) | [tcp](%stream-applications-doc%/#spring-cloud-stream-modules-tcp-sink)                                       |
+| [tcp](%stream-applications-doc%/#spring-cloud-stream-modules-tcp-source)                       | [tcp-client](%stream-applications-doc%/#spring-cloud-stream-modules-tcp-client-processor)                                   | [throughput](%stream-applications-doc%/#spring-cloud-stream-modules-throughput-sink)                         |
+| [tcp-client](%stream-applications-doc%/#spring-cloud-stream-modules-tcp-client-source)         | [tensorflow](%stream-applications-doc%/#spring-cloud-stream-modules-tensorflow-processor)                                   | [websocket](%stream-applications-doc%/#spring-cloud-stream-modules-websocket-sink)                           |
+| [time](%stream-applications-doc%/#spring-cloud-stream-modules-time-source)                     | [transform](%stream-applications-doc%/#spring-cloud-stream-modules-transform-processor)                                     |                                                                                                              |
+| [trigger](%stream-applications-doc%/#spring-cloud-stream-modules-trigger-source)               | [twitter-sentiment](%stream-applications-doc%/#spring-cloud-stream-modules-twitter-sentiment-processor)                     |                                                                                                              |
+| [triggertask](%stream-applications-doc%/#spring-cloud-stream-modules-triggertask-source)       |                                                                                                                             |                                                                                                              |
+| [twitterstream](%stream-applications-doc%/#spring-cloud-stream-modules-twitterstream-source)   |                                                                                                                             |                                                                                                              |
+
+## Task Applications
+
+The Spring team develops and maintains [task/batch applications](https://cloud.spring.io/spring-cloud-task-app-starters/) and publishes these applications to the [Spring public Maven repository](https://repo.spring.io/release/org/springframework/cloud/task/app/) and to [Dockerhub](https://hub.docker.com/u/springcloudtask) in accordance with a planned release schedule, normally following significant Spring Boot, Spring Cloud Task, or Spring Batch releases.
```

Comment:
We can say that the out of the box `task` apps are for `demo` purposes as one has to write the batch/task apps based on the use cases.

Reply:
I was thinking about that, but will not be entirely true when Michael adds the new batch app.